ibis/expr/tests/test_value_exprs.py

```diff
@@ -1076,6 +1076,13 @@ def test_time_invalid_compare_on_py2():
     assert not result
 
 
+def test_scalar_parameter_list():
+    value = ibis.param([dt.int64, dt.int32])
+
+    assert isinstance(value.op(), ir.ParamList)
+    assert value.type().equals(dt.int64)
```

Comment:
This interface feels a bit confusing to me. Why am I allowed to pass multiple types in a list?

Reply:
Agree, but I didn't have a better API in mind. 

Actually we might use `ibis.param(dt.Array(dt.int64))` instead. 
I'm not sure which one could better fit with `IN` clauses.

I'm more than open for creating a better API.

Reply:
@cpcloud ping

Reply:
Hm, I think we need something like this, but I'm wondering if something like a `Set` type is the way to go long-term. Is this feature blocking you at the moment?

`Contains` would now look a bit different with a `Set` type.

```python
class Contains:
    input_type = [rules.value, rules.set(rules.value)]
```

then for params we have `ibis.param({dt.int64})` as the API (could also allow `list`s there as well).

We could probably even get rid of `ValueList` altogether and just have `Set`s and `array` types.